Urlopik/src/main/Counter.java

```diff
@@ -11,22 +12,22 @@
 	private int total, planned, used, remaining;
 	
 			
-	public Counter(Map<LocalDate, Integer> yearMap, int total) {
+	public Counter(Map<LocalDate, Enum> yearMap, int total) {

```

Comment:
Map<LocalDate, Enum> zamień na Map<LocalDate, OffMode> to Enum jest zbyt generyczne i nie zwiększa czytelności :) a tak będzie wiadomo że mapujemy date na OffMode